packages/gatsby-plugin-screenshot/chrome/buildChrome.sh

```diff
@@ -0,0 +1,35 @@
+# build headless chrome on EC2
+# https://github.com/adieuadieu/serverless-chrome/blob/master/chrome/README.md
+
+# sudo su
+
+yum install -y git redhat-lsb python bzip2 tar pkgconfig atk-devel alsa-lib-devel bison binutils brlapi-devel bluez-libs-devel bzip2-devel cairo-devel cups-devel dbus-devel dbus-glib-devel expat-devel fontconfig-devel freetype-devel gcc-c++ GConf2-devel glib2-devel glibc.i686 gperf glib2-devel gtk2-devel gtk3-devel java-1.*.0-openjdk-devel libatomic libcap-devel libffi-devel libgcc.i686 libgnome-keyring-devel libjpeg-devel libstdc++.i686 libX11-devel libXScrnSaver-devel libXtst-devel libxkbcommon-x11-devel ncurses-compat-libs nspr-devel nss-devel pam-devel pango-devel pciutils-devel pulseaudio-libs-devel zlib.i686 httpd mod_ssl php php-cli python-psutil wdiff --enablerepo=epel
```

Comment:
Who uses this script?

Reply:
This script is for building a new version of Chrome that's compatible with AWS Lambda; see here for more information: https://github.com/sambaiz/puppeteer-lambda-starter-kit#build-headless-chrome-optional. At the moment a precompiled binary is included in this PR.